cnames_active.js

```diff
@@ -2117,6 +2117,7 @@ var cnames_active = {
   "pad": "ebraminio.github.io/pad.js", // noCF? (don´t add this in a new PR)
   "pad-project": "adaltas.github.io/node-pad-docs",
   "padmemesamidala": "padmemesamidala.github.io/PadmemesAmidalaWeb",
+  "pagelike": "https://iamanishroy.github.io/pagelike",
```

Comment:
```suggestion
  "pagelike": "iamanishroy.github.io/pagelike",
```
You need to remove the `https://`